opa-filter-core/src/main/java/com/bisnode/opa/spring/security/filter/Decision.java

```diff
@@ -0,0 +1,28 @@
+package com.bisnode.opa.spring.security.filter;
+
+import static java.lang.Boolean.TRUE;
+
+class Decision {
+    private Boolean allow;
+    private String reason;
+
+    boolean isAllow() {
+        return TRUE.equals(allow);
+    }
+
+    public Boolean getAllow() {
```

Comment:
Should we perhaps consider `@Nullable` for this library?

Reply:
I guess so. Do you have any preference in `javax.annotation` vs `jetbrains.annotations`?

Reply:
The `javax` one. I noticed there was even on included in the Spring framework which would leave us one less dependency, but sadly it doesn't seem to be supported [outside of Spring](https://github.com/JetBrains/kotlin/blob/master/core/descriptors.jvm/src/org/jetbrains/kotlin/load/java/JvmAnnotationNames.kt).